Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/IngestEventsListener.java

```diff
@@ -72,65 +72,104 @@ public void uninstallListeners() {
         IngestManager.getInstance().removeIngestJobEventListener(pcl2);
     }
 
+    /**
+     * Enable this IngestEventsListener to add contents to the Central
+     * Repository.
+     *
+     */
+    public synchronized static void enableCentralRepositoryModule() {
+        ingestJobCounter++;  //Should be called once in the central repository module's startup method.
+    }
+
+    /**
+     * Disable this IngestEventsListener from adding contents to the Central
+     * Repository.
+     */
+    public synchronized static void disableCentralRepositoryModule() {
+        if (isCentralRepositoryModuleEnabled()) {  //prevent it ingestJobCounter from going negative
+            ingestJobCounter--;  //Should be called once in the central repository module's shutdown method.
+        }
+    }
+
+    /**
+     * Reset the counter which keeps track of if the Central Repository Module
+     * is being run during injest to 0.
+     */
+    synchronized static void resetIngestJobCounter() {
+        ingestJobCounter = 0;  //called when a case is opened in case for some reason counter was not reset
+    }
+
+    /**
+     * Wether or not the Central Repository Module is enabled for any of the
+     * currently running ingest jobs.
+     *
+     * @return boolean True for Central Repo enabled, False for disabled
+     */
+    private synchronized static boolean isCentralRepositoryModuleEnabled() {
+        return ingestJobCounter > 0;
+    }
+
```

Comment:
As above, I think it would be more clear for future maintainers if we named these method to reflect that this is a counter for ingest module instances, not ingest jobs.